.rubocop_todo.yml

```diff
@@ -145,6 +145,7 @@ RSpec/FilePath:
     - 'spec/ios_merge_translators_strings_spec.rb'
     - 'spec/release_notes_helper_spec.rb'
     - 'spec/android_localize_helper_spec.rb'
+    - 'spec/configure_helper_spec.rb'
```

Comment:
This will be superseded by #254, but I didn't know about that PR at the time of writing this.

Reply:
I did rebase, but the diff is still here 🤔 

Regardless, other manual RuboCop fixes I did in this branch are now gone 💪